### DISCUSSION TITLE
Use html file to PDF

### DISCUSSION THREAD

**Discussion by KevinYangMay** [simonhaenisch/md-to-pdf#174] [Ideas]
Today,I use html file to PDF.
It's success! so cool!

**Comment by simonhaenisch** [comment]
Assuming you mean that you can convert HTML to PDF as well, not just markdown? That should be right, because it's valid to put HTML inside markdown, at least with [Marked](https://marked.js.org/).

Could mention this in the readme as a feature I guess.
